lib/SILOptimizer/Transforms/AllocBoxToStack.cpp

```diff
@@ -142,8 +143,9 @@ static bool addLastRelease(SILValue V, SILBasicBlock *BB,
 // loop.
 static bool getFinalReleases(SILValue Box,
                              SmallVectorImpl<SILInstruction *> &Releases) {
-  SmallPtrSet<SILBasicBlock *, 16> LiveIn;
-  SmallPtrSet<SILBasicBlock *, 16> UseBlocks;
+  SILFunction *function = Box->getFunction();
+  BasicBlockSetVector<16> LiveIn(function);
```

Comment:
Why did you make this semantic change? I guess we were iterating over a SmallPtrSet? So you switched to a BasicBlockSetVector?

Reply:
yes, exactly